docs/usage/faq.md

```diff
@@ -39,6 +39,17 @@ The Renovate team will only create bugfixes for an older version if the hosted a
 If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
 If you're self hosting Renovate, use the latest release if possible.
 
+## Supported ways of using Renovate
+
+Renovate is intended to be used as an app.
+We do not support using Renovate as a library.
+
+Supported ways to use Renovate:
+
+- via the CLI program
+- via the official hosted app
+- via selfhosting the Renovate app
+
```

Comment:
@rarkins Do you want to keep this section in the FAQ? I think it's a bit superfluous now that we have a full getting started guide with recommended ways to use Renovate in it.

Or we could just link to the new getting started guide?

Reply:
Yes, we can remove it. Most of the FAQ hasn't been updated in a very long time!

Reply:
```suggestion
```